themes/kiali/layouts/partials/home/features-section.html

```diff
@@ -47,7 +47,7 @@ <h2><img src="img/wizards.svg" alt="" />Wizards</h2>
       <a href="{{ "documentation/features/#_configuration_and_validation_features" | relURL }}">
         <div>
           <h2><img src="img/configuration.svg" alt="" />Configuration</h2>
-          <p>See and edit Istio<br />CRD YAML</p>
+          <p>See and edit Istio<br />resources</p>
```

Comment:
Can I suggest `edit Istio setup` or `edit Istio configuration`?
*Resources* is technically correct, but I see the resources as a way to configure/setup Istio.

Just a suggestion. You choose the one you like :)